sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaRecordCoder.java

```diff
@@ -66,9 +76,34 @@ public void encode(KafkaRecord<K, V> value, OutputStream outStream) throws IOExc
         longCoder.decode(inStream),
         longCoder.decode(inStream),
         KafkaTimestampType.forOrdinal(intCoder.decode(inStream)),
+        (Headers) toHeaders(headerCoder.decode(inStream)),
         kvCoder.decode(inStream));
   }
 
+  private Object toHeaders(Iterable<KV<String, byte[]>> records) {
+    if (!records.iterator().hasNext()){
+      return null;
+    }
+
+    // ConsumerRecord is used to simply create a list of headers
+    ConsumerRecord<String, String> consumerRecord = new ConsumerRecord<>(
```

Comment:
```Headers headers = ... ```
raises a ```ClassNotFoundException``` when serializing ```KafkaRecordCoder```.

Updated to use empty strings instead of "key", "value"